website/docs/r/project.html.markdown

```diff
@@ -79,6 +79,8 @@ api_keys allows one to assign an existing organization programmatic API key to a
 
 * `api_key_id` - (Required) The unique identifier of the Programmatic API key you want to associate with the Project.  The Programmatic API key and Project must share the same parent organization.  Note: this is not the `publicKey` of the Programmatic API key but the `id` of the key. See [Programmatic API Keys](https://docs.atlas.mongodb.com/reference/api/apiKeys/) for more.
 
+**WARNING:** This parameter is deprecated, use `mongodbatlas_project_api_key`  will be removed in v1.12.0 release
```

Comment:
```suggestion
**WARNING:** The `api_keys` parameter is deprecated and will be removed in v1.12.0 release from codebase. Use `mongodbatlas_project_api_key`  resource instead. 
```

Reply:
@Zuhairahmed this will need updating on the project data sources as well.   Maybe do as a follow up?   I'm going to approve this to unblock.